web/src/components/PoiListItem.tsx

```diff
@@ -74,16 +52,31 @@ const PoiListItem = ({ poi, distance, selectPoi }: PoiListItemProps): ReactEleme
 
   return (
     <>
-      <ListItemContainer id={slug}>
-        <LinkContainer onClick={selectPoi} tabIndex={0} label={title}>
-          <Thumbnail alt='' src={thumbnail || PoiThumbnailPlaceholder} />
-          <Description>
-            <Title>{title}</Title>
-            {distance !== null && <Distance>{t('distanceKilometre', { distance: distance.toFixed(1) })}</Distance>}
-            <Category>{category.name}</Category>
-          </Description>
-        </LinkContainer>
-      </ListItemContainer>
+      <ListItem disablePadding>
+        <StyledListItemButton onClick={selectPoi} id={slug} to={slug} component={Link} aria-label={title}>
+          <StyledListItemAvatar>
+            <Avatar src={thumbnail || PoiThumbnailPlaceholder} alt='' variant='square' />
+          </StyledListItemAvatar>
+          <StyledListItemText
+            slotProps={{
+              secondary: {
+                component: 'div',
```

Comment:
added because of the console error `In HTML, <p> cannot be a descendant of <p>.
This will cause a hydration error.`